content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+date: 2019-11-13T11:23:04-06:00
+draft: false
+meta_desc: "Pulumi supports .NET Core languages for Infrastructure as Code"
+meta_image: "meta.png"
+authors:
+    - sophia-parafina
+tags:
+    - ".Net Core"
+    - "C#"
+    - "F#"
+    - "Visual Basic"
+---
+
+We are excited to announce the preview of Pulumi .NET support. Pulumi lets you declare infrastructure using general purpose programming languages. Our goal is to automate applications at all layers and help developers and operators work better together by eliminating the silos that separate them.
+
+Millions of .NET developers can now use their favorite languages and open source ecosystems to build modern, cloud native applications. We've added support for C#, F#, and Visual Basic. Because .NET Core is available on Windows, Linux, and macOS, you have a choice of platforms to use. You can create cloud resources by writing Microsoft .NET Core programs to build and deploy AKS Clusters, Functions, Azure App Services, Virtual Machines, Cosmos DBs, KeyVaults, and much, much more.
+
+## What Does Pulumi Enable?
+
+With Pulumi, .NET developers can declare infrastructure using C#, F#, or Visual Basic .NET. This results in increased productivity for developers because they can use the features of their IDEs such as auto-completion, error checking with red markers, build error messages, refactoring tools, and package managers. Developers can reference any NuGet library compatible with .NET Core 3.0.
```

Comment:
```suggestion
With Pulumi, .NET developers can create, deploy and manage cloud infrastructure using C#, F#, or Visual Basic .NET. This results in increased productivity for developers because they can use the features of their IDEs such as auto-completion, error checking with red markers, build error messages, refactoring tools, and package managers. Developers can reference any .NET libraries on NuGet compatible with .NET Core 3.0.
```